dev-client/__tests__/context/GeospatialContext.test.tsx

```diff
@@ -27,9 +27,11 @@ const TestComponent = ({mock}: Props) => {
   const {siteDistances} = useGeospatialContext();
   useEffect(() => {
     mock(
-      Object.entries(siteDistances || {})
-        .sort(([, distanceA], [, distanceB]) => distanceA - distanceB)
-        .map(([name]) => name),
+      siteDistances === null
+        ? null
+        : Object.entries(siteDistances)
+            .sort(([, distanceA], [, distanceB]) => distanceA - distanceB)
+            .map(([name]) => name),
```

Comment:
This really should have been a different PR. It's just fixing a test I made fail in the previous PR.